gateway-service/src/main/resources/application.yml

```
@@ -346,6 +346,7 @@ server:
         scheme: http
 apiml:
     service:
+        corsEnabled: true
         scheme: http
```

```suggestion
        scheme: http
eureka:
    instance:
        metadata-map:
            apiml:
                corsAllowedOrigins: https://${apiml.service.hostname}:${apiml.service.port}
```

I guess it should be enough, but I don't know if you tested it yet or no...

---

why here? it's added already in catalog

---

I still don't think it is necessary